tests/unit_tests/dynamics/collisions/test_fragmentations.py

```diff
@@ -222,51 +228,94 @@ def test_fragmentation_limiters_nfmax(
 
     @staticmethod
     @pytest.mark.parametrize(
-        "fragmentation_fn, volume, expected_nf",
+        "fragmentation_fn",
         (
-            (
-                ConstantSize(c=4 * si.um**3),
-                np.asarray([400.0 * si.um**3, 600.0 * si.um**3]),
-                250,
-            ),
-            (AlwaysN(n=250), np.asarray([400.0 * si.um**3, 600.0 * si.um**3]), 250),
+            AlwaysN(n=2),
+            ExponFrag(scale=1e6 * si.um**3),
+            Feingold1988Frag(scale=1e6 * si.um**3),
+            Gaussian(mu=2e6 * si.um**3, sigma=1e6 * si.um**3),
+            SLAMS(),
+            Straub2010Nf(),
         ),
     )
-    def test_fragmentation_nf_and_frag_size_equals(  # TODO #987
-        fragmentation_fn,
-        volume,
-        expected_nf,
-        backend_class=CPU,  # pylint:disable=redefined-outer-name
-    ):
+    def test_fragmentation_fn_distribution(
```

Comment:
@edejong-caltech this change removes `test_fragmentation_nf_and_frag_size_equals` and drops test coverage for `ConstantSize` fragmentation function. Was it intentional?